### PR TITLE
Staging deployment

### PR DESCRIPTION
**Pull request by Xhark-Blues**

### CODE REVIEW

File: k8s.production.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      containers:
+        - name: muninn
+          image: registry.repositive.io:5000/muninn:${TAG}
+          env:
+            - name: BROADCAST_URI
+              valueFrom:
+                secretKeyRef:
+                  name: rabbit-urls-production
+                  key: rabbit
+            - name: DB_URI
```
Comment (hectorhuertas) [inline k8s.production.yml]: It looks like we dont need the db_uri var

Comment (hectorhuertas) [reply k8s.production.yml]: for redis we can have one with the format
```
- name: REDIS_GODS_URI
  value: redis://redis:6379/4
```

**Review by hectorhuertas — CHANGES_REQUESTED** [review]

**Review by hectorhuertas — APPROVED** [review]
LGTM

**Review by Istar-Eldritch — APPROVED** [review]
Looks good! :rocket:

**Pull request merged**